Apps/W1/HybridGP/app/src/Migration/Support/HelperFunctions.codeunit.al

```diff
@@ -865,10 +865,10 @@ Codeunit 4037 "Helper Functions"
                 if GPPMHist.FINDFIRST() then
                     GPPMHist.MODIFYALL(GPPMHist."PYMTRMID", PaymentTerm_New);
 
-                GPPOPPOHeader.Reset();
-                GPPOPPOHeader.SetRange(GPPOPPOHeader.PYMTRMID, PaymentTerm);
-                if GPPOPPOHeader.FindFirst() then
-                    GPPOPPOHeader.ModifyAll(GPPOPPOHeader.PYMTRMID, PaymentTerm_New);
+                GPPOP10100.Reset();
+                GPPOP10100.SetRange(GPPOP10100.PYMTRMID, PaymentTerm);
+                if GPPOP10100.FindFirst() then
```

Comment:
Performance-wise, an if not IsEmpty() pattern would be more efficient.

Reply:
Agreed, updated. Thanks